app/decorators/menu_decorator.rb

```diff
@@ -0,0 +1,8 @@
+class MenuDecorator < Draper::Decorator
+  delegate_all
+
+  def day_stamp
+    "#{created_at.strftime('%A')}, #{created_at.day}th of "\
```

Comment:
Just FYI: `I18n.locale` can help you.
[source](http://guides.rubyonrails.org/i18n.html#adding-date-time-formats)

Reply:
https://apidock.com/rails/ActiveSupport/Inflector/ordinalize